lib/local_participant_view.dart

```diff
@@ -0,0 +1,130 @@
+// © 2022 Daily, Co. All Rights Reserved
+
+import 'dart:async';
+
+import 'package:daily_flutter/daily_flutter.dart';
+import 'package:daily_flutter_demo/user_settings_bottom_sheet.dart';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class LocalParticipantView extends StatefulWidget {
+  const LocalParticipantView({super.key, required this.client, required this.prefs});
+
+  final CallClient client;
+  final SharedPreferences prefs;
+
+  @override
+  State<LocalParticipantView> createState() => _LocalParticipantViewState();
+}
+
+class _LocalParticipantViewState extends State<LocalParticipantView> {
+  StreamSubscription? _eventSubscription;
+  final _controller = VideoViewController();
+  late String _username = widget.client.participants.local.info.username ?? 'Guest';
```

Comment:
Is it worth also setting to "Guest" if the username is an empty string? Or is that not a possibility here for reasons I'm unfamiliar with?